README.md

```diff
@@ -0,0 +1,4 @@
+# Cromwell
+Cromwell is a replacement firmware for the original Xbox . It can boot the Linux operating system and practically convert the Xbox into a full PC.
```

Comment:
```suggestion
Cromwell is a replacement firmware for the original Xbox. It can boot the Linux operating system and practically convert the Xbox into a full PC.
```